src/csharp/Microsoft.Spark/Interop/Internal/Dotnet/Dictionary.cs

```diff
@@ -0,0 +1,44 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Spark.Interop;
+using Microsoft.Spark.Interop.Internal.Java.Util;
+
+namespace System.Collections.Generic
+{
+    public static class Dictionary
+    {
+        /// <summary>
+        /// A custom extension method that helps transform from dotnet
+        /// Dictionary&lt;string, string&gt; to java.util.HashMap.
+        /// </summary>
+        /// <param name="value">a Dictionary instance</param>
+        /// <returns><see cref="HashMap"/></returns>
+        internal static HashMap ToHashMap(this Dictionary<string, string> value)
```

Comment:
I don't see either of these functions referenced anywhere, is this to be used for future PRs?

Reply:
This is needed for future usage, like when we incorporate more spark.ml features.